### ISSUE TITLE
Onboarding Checklist for ES New Editor: Joshua Ortiz Baco

### ISSUE THREAD

**Issue by jenniferisasi** [programminghistorian/jekyll#1191]
This is an individual onboarding editor checklist for Joshua Ortiz Baco (@joshuaGOB), initially found [here](https://github.com/programminghistorian/jekyll/wiki/Onboarding-Checklist-for-New-Editors). 

We've put together this information to help you get up to speed. It includes details of what we will do and a checklist of tasks you should complete, to make sure you know how to perform key tasks. 

## What we will do
Within 2 weeks of your joining, our current team will ensure that you have been granted access to the following services used by the Project Team:

- [x] Our Google Group email so you can communicate with other editors (@acrymble)
- [x] Our Google Analytics account for monitoring web traffic (@acrymble)
- [x] Our Twitter team (@jenniferisasi )
- [x] Our Github Repository so you can upload materials to the site (@mdlincoln)
- [x] Our Skype group so that you can participate in our monthly calls (@amsichani)
- [x] We will also write a blog post welcoming you publicly to the team.
- [ ] partner you with an existing editor to 'shadow' the editing process so you can get a feel for it.

## What you should do
There are a few tasks you will need to perform to make sure you know how to use our system, and also to make sure your details appear on the Project Team page. All of these tasks are encouraged, but some may be optional:

### communication
- [x] set up a GitHub account and inform @mdlincoln of your username (mandatory)
- [x] set up a twitter account (optional)
- [x] set up a Skype account and inform @amsichani (mandatory)
- [x] inform @acrymble of your Google-activated email (mandatory)
- [x] write an email to the other editors using the Google group (say hello!) (mandatory)

### technical
We rely heavily on Github for communicating and conducting our editorial processes. You will need to get familiar with it and the way we use it. We recognise that this can take time, so if you would like one-on-one training, please let someone know and this will be arranged. To make sure you can do all required tasks, try the following activities on our Github /jekyll website.

- [x] contribute to one open ticket on our github repository (mandatory)
- [x] create and assign to yourself a new ticket on the github repository to ensure you know how to do this. Add a relevant label. (mandatory)
- [x] make a technical contribution on your own. This will normally be adding your bio to the /data/ph_authors.yml file, and uploading a square black and white avatar (.png) of yourself to /avatars). You will then submit a pull request for approval so that this bio information can be added to the Project Team page (mandatory)
- [x] Read all sections of the /contribute pages of the website, including author, reviewer, and editor guidelines
- [x] Bookmark the /jekyll (https://github.com/programminghistorian/jekyll) and /submissions (https://github.com/programminghistorian/ph-submissions) github sites so you can find them easily
- [x] Read the 'Making Technical Contributions' wiki page: https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions

### The Culture of the Project
We have invited you to join the team because we value your opinions. We encourage all of our editors to contribute to discussions on github or via email, and to civilly express their views, even if they disagree with any or all other members. This right to express one's opinions is central to our project. Please take this to heart.

**Comment by acrymble** [comment]
Hi Joshua, Can you please send me an email with your google-enabled email (adam.crymble@gmail.com) so I can set you up?

**Comment by acrymble** [comment]
@jenniferisasi can you follow up with Joshua to see if he's having trouble?

**Comment by JoshuaGOB** [comment]
Hello all,

skype: live:joshuaortizbaco @amsichani 
gmail: joshuaortizbaco@gmail.com @acrymble

**Comment by acrymble** [comment]
@jenniferisasi Is there a plan in place to get @JoshuaGOB and @rivaquiroga shadowing on a lesson? I suspect neither of them really understands how to get involved yet.

**Comment by acrymble** [comment]
@JoshuaGOB I'm not sure if you've managed to find a lesson to shadow yet. If so, can we close this? If not, can you let us know and we can find you an opportunity?

**Comment by JoshuaGOB** [comment]
Hi, thank you for checking in. I haven’t been able to yet. @jenniferisasi is there one around?

**Comment by jenniferisasi** [comment]
> @jenniferisasi Is there a plan in place to get @JoshuaGOB and @rivaquiroga shadowing on a lesson? I suspect neither of them really understands how to get involved yet.

@acrymble there is currently no lesson/translation process where we could put either of them on shadowing mode from the beginning, so we are waiting on a new lesson however @JoshuaGOB is willing to shadow a lesson with the English team as you are receiving a few of them. 

If you want, we can close the ticket as I'm not forgetting this missing task.

**Comment by acrymble** [comment]
Sure @JoshuaGOB I am about to publish an English lesson. Did you want to shadow me in that process so you can get a feel for it? @rivaquiroga too if you'd like?

**Comment by JoshuaGOB** [comment]
That sounds good. What’s the schedule like? I only ask because I have too grant apps due this week and a conference the week after.

**Comment by acrymble** [comment]
I suspect this week or next.

**Comment by JoshuaGOB** [comment]
Perfect. Let me know.

**Comment by acrymble** [comment]
@JoshuaGOB is shadowing me as I publish a lesson. I'll close this on that basis, but we'll also make sure he gets a more substantial opportunity to shadow on editing.
